spec/rails_helper.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+require 'coveralls'
+Coveralls.wear_merged!('rails')
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'rspec/rails'
```

Comment:
can you put space after this line please

Reply:
Done